### PR TITLE
updated personal access token generation instructions

### PR DESCRIPTION
**Pull request by oindrillac**
## Related Issues and Dependencies

resolves https://github.com/aicoe-aiops/ocp-ci-analysis/issues/558
Added automated generation of PAT.

## This introduces a breaking change

- [ ] Yes
- [x] No

cc: @schwesig thanks for the suggestion 👏

### CODE REVIEW

File: docs/workshop/git_setup.md [hunk 1]
```diff
@@ -2,14 +2,15 @@
 
 In order to get started with the workshop, it's vital to have a github account, so you can also interact with the codebase and make changes as and when required. Here are a few steps that you need to follow along to be all set to get started with this workshop.
 
-* **Create a Github account**: Go to[ github.com](https://github.com/) and click on `Sign Up` to create a new account.
 
-![alt_text](../assets/images/git-setup-git-account.png "image_tooltip")
-
-* **Fork the repository:** After creating your account, you need to access the AI4CI repository to interact with the code. In order to do so, go to the [AI4CI repository](https://github.com/aicoe-aiops/ocp-ci-analysis) and fork the repository by clicking on the `fork` button In the top-right corner of the page.
+* **Fork the repository:** You should already have a Github account (if not refer to [onboarding](./onbaording.md), you need to access the AI4CI repository to interact with the code. In order to do so, go to the [AI4CI repository](https://github.com/aicoe-aiops/ocp-ci-analysis) and fork the repository by clicking on the `fork` button In the top-right corner of the page.
```
Comment (suppathak) [inline docs/workshop/git_setup.md:6]:
```suggestion
* **Fork the repository:** You should already have a Github account (if not refer to [onboarding](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/docs/workshop/onboarding.md), you need to access the AI4CI repository to interact with the code. In order to do so, go to the [AI4CI repository](https://github.com/aicoe-aiops/ocp-ci-analysis) and fork the repository by clicking on the `fork` button In the top-right corner of the page.
```
`Onboarding` link not working.

**Review by suppathak — COMMENTED** [review]
Thanks @oindrillac . Just made a small comment on broken link.

File: docs/workshop/git_setup.md (hunk 1 above)
Comment (oindrillac) [reply docs/workshop/git_setup.md:6]: Thanks for catching this! thats coz I misspelled it haha. I will fix that. Lets keep the link relative

Comment (oindrillac) [reply docs/workshop/git_setup.md]: @suppathak made the change. PTAL

**Review by suppathak — COMMENTED** [review]
Thanks @oindrillac .
/lgtm

**Pull request merged**